client/src/App.js

```diff
@@ -20,6 +20,7 @@ const App = function() {
 
   //API Call to Retrieve All Beverages
   useEffect(() => {
+    console.log(`${process.env.REACT_APP_DEV_API_URL}`);
```

Comment:
Forget to remove your console.log?? :stuck_out_tongue:

Reply:
Ah, let me get that one two, I just removed another one